src/styles/blocks/button.scss

```
@@ -0,0 +1,77 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  width: 100%;
+  margin-bottom: 30px;
+  @include onTablet {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  background-color: #cd4d31;
+
```

Change order here and add page brakes all over the scss files.
```suggestion
  width: 100%;
  margin-bottom: 30px;
  background-color: #cd4d31;
  
  @include onTablet {
    width: 100%;
    margin-bottom: 40px;
  }

```